### ISSUE TITLE
Terminating TLS at a reverse proxy doesn't work

### ISSUE THREAD

**Issue by strugee** [pump-io/pump.io#1239]
Ideally when reverse proxying you'd terminate TLS at the proxy layer and send pump.io plain HTTP. However, this doesn't seem to work.

**Comment by ian-kelling** [comment]
Note: Apache can't do ssl passthrough, you need nginx or haproxy.

**Comment by strugee** [comment]
Yeah, the title is a little misleading since what I end up telling people to do is still terminate HTTPS at the Apache layer, but then start a new TLS session with the backend process. Which is usually on localhost. :/

That's pretty unfortunate, so that's what this issue is about.
